ModelicaTest/Resources/Data/Tables/test.csv

```diff
@@ -0,0 +1,3 @@
+time, value
+0.0, 0.0	
+1,1
```

Comment:
Why is there no end-of-line on the last line?
I see some problems with handling such files - and I certainly don't want to make that mandatory.

Reply:
This was taken from the original test file provided by @dietmarw. Both cases, that is with and w/o trailing new-line are covered. If you prefer, I can add it here, too.

Reply:
There are now also test models in ModelicaTest that refere to CSV files with trailing new-line. Please review again.